google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -159,14 +160,19 @@ public void sendEvent(
           eventMetadataMap.put(METADATA_ESCAPER.escape(eventLabel), eventValue);
           fullMetadataString = fullMetadataString + "," + METADATA_JOINER.join(eventMetadataMap);
         }
+        if (eventMessage != null) {
+          Map<String, String> eventMessageMap =
+              ImmutableMap.of("Message", METADATA_ESCAPER.escape(eventMessage));
```

Comment:
@chanseokoh what do you think of this key value mapping? I'm just hardcoding the key to "Message" here.